src/cnfizers/Cnfizer.cc

```diff
@@ -210,9 +211,4 @@ void Cnfizer::Cache::insert(PTRef term, FrameId frame) {
     assert(!contains(term, frame));
     cache.insert({term, frame});
 }
-
-// TODO: Fix isAtom!!! (everything that is not a boolean connective should be considered as atom
-bool Cnfizer::isLiteral(PTRef ptr) const {
-    return (logic.isNot(ptr) and logic.isAtom(logic.getPterm(ptr)[0])) or logic.isAtom(ptr);
-}
```

Comment:
Why not keep this function, just change its implementation to `return logic.isLiteral(ptr);`?
Nothing else in this file needs to change then, no?